packages/browsers/src/CLI.ts

```diff
@@ -396,6 +400,45 @@ export class CLI {
           );
         },
       )
+      .command(
+        'list',
+        'List all installed browsers in the cache directory',
+        yargs => {
+          this.#definePathParameter(yargs);
+          yargs.example(
+            '$0 list',
+            'List all installed browsers in the cache directory',
+          );
+          if (this.#allowCachePathOverride) {
+            yargs.example(
+              '$0 list --path /tmp/my-browser-cache',
+              'List browsers installed in the specified cache directory',
+            );
+          }
+        },
+        async argv => {
+          const args = argv as unknown as ListArgs;
+          const cacheDir = args.path ?? this.#cachePath;
+
+          try {
+            const cache = new Cache(cacheDir);
+            const browsers = cache.getInstalledBrowsers();
+
+            if (browsers.length === 0) {
+              console.log('No browsers installed.');
```

Comment:
I wonder if this should return nothing to indicate that there are no browsers installed. I would help to use the command output with other tools.